packages/lit-dev-server/src/middleware/content-security-policy-middleware.ts

```diff
@@ -89,9 +89,6 @@ export const contentSecurityPolicyMiddleware = (
     // es-module-lexer for doing an eval (see next comment for more on that).
     `script-src ${[
       `'self'`,
-      // TODO(aomarks) Remove unsafe-eval when https://crbug.com/1253267 is fixed.
-      // See comment below about playgroundWorkerCsp.
-      `'unsafe-eval'`,
```

Comment:
No action required - but this change could be broken into an independent PR - mostly if something strange happened it could be rolled back in isolation.

Reply:
Good idea. Split this out here: https://github.com/lit/lit.dev/pull/1130